cel/io.go

```diff
@@ -59,14 +74,25 @@ func AstToCheckedExpr(a *Ast) (*exprpb.CheckedExpr, error) {
 
 // ParsedExprToAst converts a parsed expression proto message to an Ast.
 func ParsedExprToAst(parsedExpr *exprpb.ParsedExpr) *Ast {
+	return ParsedExprToAstWithSource(parsedExpr, nil)
+}
+
+// ParsedExprToAstWithSource converts a parsed expression proto message to an Ast,
```

Comment:
Please rephrase in a similar manner to the `CheckedExprToAstWithSource`. The use case on this method is actually really difficult for me to understand, but I understand the desire for functional parity.

Perhaps recommend only using this function if you need to re-check a previously type-checked expression?